perfrunbook/utilities/capture_flamegraphs.sh

```diff
@@ -3,14 +3,15 @@
 set -e
 
 let capture_freq=99
+unset reverse
```

Comment:
What is this for? How would reverse be set given there's nothing above this?

Reply:
Not sure I can remove it, but it ensures `reverse` is in an unset state for the below `${reverse:+--reverse}` expansion to conditionally expand if the reverse variable is set with a value.